lib/pghero/methods/indexes.rb

```diff
@@ -539,140 +539,444 @@ def duplicate_indexes(indexes: nil)
+            btree_index_atts AS (
+              SELECT
+                nspname, relname, reltuples, relpages, indrelid, relam,
+                regexp_split_to_table(indkey::text, ' ')::smallint AS attnum,
+                indexrelid as index_oid
+              FROM
+                pg_index
+              JOIN (
+                SELECT
+                  c.oid,
+                  relnamespace,
+                  relam,
+                  di.indexname AS relname,
+                  sum(di.reltuples) AS reltuples,
+                  sum(di.relpages) AS relpages
+                FROM
+                  dist_index_reltuples_relpages di
+                JOIN
+                  dist_indexes d ON di.indexname = d.indexname
+                JOIN
+                  pg_class c ON (schemaname || '.' || di.indexname)::regclass::oid = c.oid
+                GROUP BY
+                  4, 1, 3, 2
+                UNION ALL
+                SELECT
+                  oid,
+                  relnamespace,
+                  relam,
+                  relname,
+                  reltuples,
+                  relpages
+                FROM
+                  pg_class c
+                WHERE
+                  NOT EXISTS(SELECT * FROM dist_indexes WHERE (schemaname || '.' || indexname)::regclass::oid = c.oid)
+              ) pg_class_with_dist ON pg_class_with_dist.oid=pg_index.indexrelid
+              JOIN
+                pg_namespace ON pg_namespace.oid = pg_class_with_dist.relnamespace
+              JOIN
+                pg_am ON pg_class_with_dist.relam = pg_am.oid
+              WHERE
+                pg_am.amname = 'btree'
+            ),
+            pg_dist_stats_double_json AS (
+              SELECT (
+                SELECT
+                  json_agg(row_to_json(f))
+                FROM (
+                  SELECT
+                    result
+                  FROM
+                    run_command_on_placements(logicalrelid, $$
+                      SELECT
+                        json_agg(row_to_json(d))
+                      FROM (
+                        SELECT
+                          '$$ || logicalrelid || $$' AS dist_table,
+                          schemaname,
+                          attname,
+                          null_frac,
+                          reltuples,
+                          avg_width
+                        FROM
+                          pg_stats s
+                        JOIN
+                          pg_class c ON s.tablename = c.relname
+                        WHERE
+                          c.oid = '%s'::regclass::oid
+                      ) d $$)
+                ) f)
+              FROM
+                pg_dist_partition
+            ),
+            pg_dist_stats_single_json AS (
+              SELECT
+                (json_array_elements(json_agg)->>'result') AS result
+              FROM
+                pg_dist_stats_double_json
+            ),
+            pg_dist_stats_regular AS (
```

Comment:
pg_dist_stats_regular -> pg_dist_stats

also I'd prefer better names instead of single_json or double_json, but I dont have any ide right now. Please consider alternative names.